### PR TITLE
implement Accordion design in product page

### PR DESCRIPTION
**Pull request by nori3506**
- when Close
スクリーンショット 2021-11-10 19 19 35

- when Open
スクリーンショット 2021-11-10 19 19 24

### CODE REVIEW

File: client/src/screens/Product.js [hunk 1]
```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
       <Carousel
         data={product.images}
         renderItem={carouselImages}
-        itemWidth={650}
-        sliderWidth={650}
+        itemWidth={250}
+        sliderWidth={350}
```
Comment (satoshinbp) [inline client/src/screens/Product.js:84]:
```suggestion
        itemWidth="250px"
        sliderWidth="350px"
```
Just for clarity.

File: client/src/screens/Product.js [hunk 2]
```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
...
         renderItem={({ item, index }) => (
           <>
-            <Text>{item.question.description}</Text>
+            <Accordion>
+              <Accordion.Item>
+                <Accordion.Summary>
+                  <View style={{ flexDirection: 'column' }}>
+                    <Text>{item.question.description}</Text>
+                    <Text>This has 3 ansers</Text>
+                  </View>
```
Comment (satoshinbp) [inline client/src/screens/Product.js:98]: use VStack

File: client/src/screens/Product.js [hunk 3]
```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
...
+                  <Text style={{ textAlign: 'right' }}>Answer</Text>
```
Comment (satoshinbp) [inline client/src/screens/Product.js:99]: avoid using style props for native base component.

File: client/src/screens/Product.js [hunk 4]
```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
...
+                    margin: 0,
+                    padding: 0,
+                    backgroundColor:
+                      'linear-gradient(180deg, rgba(255, 200, 20, 0.52) 0%, rgba(255, 255, 255, 0.8) 85.42%);',
+                  }}
+                >
+                  {item.answers.map((answer, i) => (
+                    <>
+                      <View
+                        style={{
```
Comment (satoshinbp) [inline client/src/screens/Product.js:113]: avoid using style props for native base component

File: client/src/screens/Product.js [hunk 5]
```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
...
+                <Accordion.Summary>
+                  <View style={{ flexDirection: 'column' }}>
+                    <Text>{item.question.description}</Text>
+                    <Text>This has 3 ansers</Text>
+                  </View>
+                  <Text style={{ textAlign: 'right' }}>Answer</Text>
+                  <Accordion.Icon />
+                </Accordion.Summary>
+                <Accordion.Details
+                  style={{
```
Comment (satoshinbp) [inline client/src/screens/Product.js:103]: avoid using style props for native base component

File: client/src/screens/Product.js [hunk 6]
```diff
@@ -114,24 +150,54 @@ export default ({ navigation }) => {
...
         renderItem={({ item, index }) => (
           <>
-            <Text>{item.question}</Text>
+            <Accordion>
+              <Accordion.Item>
+                <Accordion.Summary>
+                  <View style={{ flexDirection: 'column' }}>
+                    <Text>{item.question.description}</Text>
+                    <Text>This has 3 ansers</Text>
+                  </View>
```
Comment (satoshinbp) [inline client/src/screens/Product.js:170]: use VStack

File: client/src/screens/Product.js [hunk 7]
```diff
@@ -114,24 +150,54 @@ export default ({ navigation }) => {
...
+                  <Text style={{ textAlign: 'right' }}>Answer</Text>
+                  <Accordion.Icon />
+                </Accordion.Summary>
+                <Accordion.Details
+                  style={{
+                    margin: 0,
```
Comment (satoshinbp) [inline client/src/screens/Product.js:176]: avoid using style props for native base component

File: client/src/screens/Product.js [hunk 8]
```diff
@@ -114,24 +150,54 @@ export default ({ navigation }) => {
...
+                    padding: 0,
+                    backgroundColor:
+                      'linear-gradient(180deg, rgba(255, 200, 20, 0.52) 0%, rgba(255, 255, 255, 0.8) 85.42%);',
+                  }}
+                >
+                  {item.answers.map((answer, i) => (
+                    <>
+                      <View
+                        style={{
```
Comment (satoshinbp) [inline client/src/screens/Product.js:185]: avoid using style props for native base component

**Review by satoshinbp — CHANGES_REQUESTED** [review]

File: client/src/screens/Product.js (hunk 1 above)
Comment (nori3506) [reply client/src/screens/Product.js]: it seems spec of Carousel??
"px" did not work...

**Review by satoshinbp — APPROVED** [review]

**Pull request merged**